drift/core/psestimation.py

```diff
@@ -592,31 +597,28 @@ def fisher_bias(self):
 
     # ====== Estimate the q-parameters from data ========
 
-    def q_estimator(self, mi, vec1, vec2=None, noise=False):
-        """Estimate the q-parameters from given data (see paper).
-
+    def project_vector_kl_to_sky(self, mi, vec1, vec2=None):
```

Comment:
Perhaps this routine should go in `kltransform` instead of `psestimation`?

Reply:
At the moment this function has to option to project 2 vectors from KL to sky - I wonder if it's better just to write it for a single vector. And if you do an auto power spectrum estimation just make a copy in q_estimator, when you do a cross power spectrum just call the function again....

Reply:
Changing this to projecting only one vector at a time.